x/src/cargo.rs

```diff
@@ -75,6 +77,25 @@ impl Cargo {
         self
     }
 
+    // Add a series of arguments to the x command.
```

Comment:
Could you make this a doc comment?

Also our standard for doc comments is "Adds a series of arguments" (i.e. the verb is plural).

Reply:
done.